templates/part.board.sidebarView.php

```diff
@@ -75,7 +75,7 @@
                     <div class="label-edit" ng-if="label.edit">
                         <div ng-style="{'background-color':'#{{label.color}}','color':'{{ textColor(label.color) }}','width':'100%'}>
                             <form ng-submit="labelUpdate(label)">
```

Comment:
Thanks for your contribution. 

Looks like the issue here is the missing " in line 76. Then it should work fine due to the form ng-submit and we don't require the ng-keyup.

Mind to change that?

Reply:
You are right, thanks for pointing it out. 

I was confused by the ng-submit behavior: I thought it worked only if there is a submit button, but it also work if there is only one input